js/os.ts

```diff
@@ -128,8 +128,8 @@ export function readFileSync(filename: string): Uint8Array {
   return new Uint8Array(dataArray!);
 }
 
-function createEnv(_msg: fbs.EnvironRes): { [index:string]: string } {
-  const env: { [index:string]: string } = {};
+function createEnv(_msg: fbs.EnvironRes): { [index: string]: string } {
```

Comment:
hmmm... I have prettier deal with things on save, and didn't pay attention to this reformatting, but it is valid, though not related to this PR.